django_rq/views.py

```diff
@@ -368,3 +370,30 @@ def actions(request, queue_index):
                 messages.info(request, 'You have successfully requeued %d  jobs!' % len(job_ids))
 
     return redirect('rq_jobs', queue_index)
+
+
+@staff_member_required
+def enqueue_job_view(request, queue_index, job_id):
```

Comment:
Can you rename this to `enqueue_job`?

Reply:
This would broke an existing naming scheme though, line 274 with `requeue_job_view` unless you want me to change that as well

Reply:
`requeue_job_view` is called that because `requeue_job` is already a function imported from RQ.